src/dataDisplay/IconText/index.tsx

```diff
@@ -25,11 +26,20 @@ const StyledIconText = styled.div`
 /**
  * The `IconText` renders an icon next to a text
  */
-function IconText({ iconSize, textSize, iconType, text, color }: Props) {
+function IconText({
```

Comment:
no return type, mixed usage of function/arrow function syntax

Reply:
done!